README.md

```diff
@@ -77,7 +77,9 @@ most popular Emacs packages like `company`, `flycheck` and `projectile`.
     for better discovery
 
 ## Presentations/demos
-
+  - [System Crafters](https://twitter.com/SystemCrafters) channel
+    [Build Your Own IDE with lsp-mode](https://www.youtube.com/watch?v=E-NAM9U5JYE)
+    [Emacs IDE - How to Debug Your Code with dap-mode](https://www.youtube.com/watch?v=0bilcQVSlbM)
```

Comment:
Since I'm planning to do more videos in the "Emacs IDE" series, it might be better to link to the playlist so that people can find them all easily:

```suggestion
    [Emacs IDE Videos](https://www.youtube.com/playlist?list=PLEoMzSkcN8oNvsrtk_iZSb94krGRofFjN)
```

Reply:
Sure.

Reply:
Maybe I should drop the other link as well and keep only the reference to the playlist?

Reply:
Sounds fine to me!